test/conformance/header_test.go

```diff
@@ -55,14 +57,15 @@ func TestShouldHaveHeadersSet(t *testing.T) {
 
 	expectedHeaders := map[string]*regexp.Regexp{
 		// We expect the protocol to be http for our test image.
-		"x-forwarded-proto": regexp.MustCompile("http"),
-		// We expect the value to be a list of at least one comma separated IP addresses.
-		"x-forwarded-for": regexp.MustCompile("(\\d{1,3}\\.){3}\\d{1,3}(, (\\d{1,3}\\.){3}\\d{1,3})*"),
+		"x-forwarded-proto": regexp.MustCompile("https?"),
+		// We expect the value to be a list of at least one comma separated IP addresses (IPv4 or IPv6).
+		// We do additional validation of the IP addresses below.
+		"x-forwarded-for": regexp.MustCompile("[0-9a-f.:]*(, [0-9a-f.:]*)*"),
 
 		// Trace Headers
 		// See https://github.com/openzipkin/b3-propagation#overall-process
 		// We use the multiple header variant for tracing. We do not validate the single header variant.
-		// We expect the value to be a 64-bit hex string
+		// We expect the value to be a 64-bit hex stringgg
```

Comment:
probably one `g` is enough :-)

Reply:
pro tip: g only works to move around when you are not in insert mode.